test/FIFOMarket.test.ts

```diff
@@ -91,7 +149,31 @@ describe('FIFOMarket', () => {
         });
       }
     });
+    it('correctly intializes state variables', async () => {
```

Comment:
can you add a todo here or extend this test so it tests for all initial state

`// todo test remaining initialization values` would be sufficient